style.css

```
@@ -0,0 +1,344 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --linewidth: 4rem;
+    --lineheight: 0.6rem;
+    --hamburger-gap: 0.3rem;
+    --milkapurple: #7c6aab;
+    --light-bgr: #f4f4f7;
+    --highlight-color: #7fd8f4;
+    --animation-time: 200ms ease-in-out;
+}
+
+body {
+    scroll-behavior: smooth;
+}
+
+/*        m-am blocat la meniu         */
+#hamburger-menu::before,
+#hamburger-menu::after,
+#hamburger-menu input {
+    content: "";
+    width: var(--linewidth);
+    height: var(--lineheight);
+    background-color: var(--milkapurple);
+    border-radius: 70rem;
+    transition: opacity var(--animation-time); 
+}
+
+#hamburger-menu {
+    display: flex;
+    flex-direction: column;
+    gap: var(--hamburger-gap);
+    width: max-content;
+    position: absolute;
+    inset-inline-start: 1%;
+    inset-block-start: 1% ;
+    z-index: 2;
+    cursor: pointer;
+}
+
+#hamburger-menu input {
+    appearance: none;
+    padding: 0;
+    margin: 0;
+    pointer-events: none;
+}
+
+#hamburger-menu input:checked {
+    opacity: 0;
+    width: 0;
+}
+
+.sliding-menu {
+    translate: -100%;
+    transition: translate var(--animation-time);
+    padding: 0.5rem 1rem;
+    background-color: var(--milkapurple);
+    color: var(--light-bgr);
+}
+
+.nav-wrap {
+   display: flex;
+   flex-direction: row;
+   justify-content: center ;
+    
+}
+.sliding-menu a {
+    color: #fff;
+    padding: 2rem;
+    font-family: Roboto;
+    font-weight: 600;
+    text-transform: uppercase;
+}
+/* aici s termina codul de meniu */
+body h1 {
+    font-family: Roboto;
+    font-weight: 400;
+    text-transform: uppercase;
+}
+
+.fontcursive {
+    font-family: Lobster;
+    color: #fff;
+    font-size:5rem;
+}
+
+.fontnarrow {
+    font-family: Bebas Neue;
+    color: var(--highlight-color);
+    font-size: 4rem;
+}
+
+.tagline {
+    padding: 2rem;
+}
+
+.tagline .fontcursive {
+    font-family: Lobster;
+    color: #7fd8f4;
+    font-size:5rem;
+}
+
+.tagline .fontnarrow {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: Bebas Neue;
+    color: #fff;
+    font-size: xx-large;
+    margin-top: -1rem;
+}
+
+a {
+    text-decoration: none;
+    text-transform: uppercase;
+    padding: 0.5em 1em;
+    margin-block: 1rem;
+}
+
+.hero-promo {
+    padding-block: 3rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.hero-promo img {
+    display: flex;
+    align-self: center;
+    justify-self: left;
+    max-width: 95vw;

```

Vezi dacă te ajută să pui și width
```suggestion
    max-width: 95vw;
    width: 100%;
```